src/renderer/components/reader/ReaderMenu.tsx

```diff
@@ -78,7 +78,7 @@ export class ReaderMenu extends React.Component<Props, State> {
                 disabled: !publication.LOI || publication.LOI.length === 0,
             },
             {
-                title: __("reader.marks.landmarks"),
+                title: __("reader.marks.bookmarks"),
                 content: this.createLandmarkList(),
```

Comment:
The function `this.createLandmarkList()` seems to produce a list of *bookmarks*, which is inconsistent with the function name.

Reply:
TODO in separate issue